lib/build.js

```diff
@@ -32,6 +32,7 @@ module.exports = function (options, done) {
 
   if (options.debug) args.push('--debug');
   if (options.serverOnly) args.push('--server-only');
+  if (options.verbose) args.push('--verbose');
```

Comment:
This is the line that isn't covered in the tests. Add a new test in `test/build.js` that tests that the verbose option is correctly passed to the build command. 👍 You can use this as an example: https://github.com/onmodulus/demeteorizer/blob/master/test/build.js#L68-L83